apps/artwork/components/auction/templates/bid.jade

```diff
@@ -1,7 +1,8 @@
 - myLastMaxBid = me && me.lot_standing && me.lot_standing.most_recent_bid.max_bid.cents
 - increments = sale_artwork.bid_increments.filter(function(i) { return i > (myLastMaxBid || 0) })
 - firstIncr = increments[0]
-
+- leadingBidder = me && me.lot_standing && !! me.lot_standing.is_leading_bidder
```

Comment:
Is there a reason for the `!! ` here instead of just doing `!me.lot_standing.leading_bidder`? I _think_ you may also be able to get away with the existential operator here (`me?.lot_standing?.is_leading_bidder`), although looks like it's not used above. Is there a style preference @craigspaeth ?